src/en/space-station-14/departments/service/proposals/gnome-ghost-role.md

```diff
@@ -0,0 +1,85 @@
+# Botany helper gnomes
+
+| Designers      | Implemented | GitHub Links | Sprites |
+| -------------- | ----------- | ------------ |---------
+| BITTERLYNX     | :x: No      | https://github.com/space-wizards/space-station-14/pull/26941  | DanOftheE |
+
+# Botany helper gnomes
+
+Have YOU ever needed to garden AND needed to grab a drink at the bar? 
+Have YOU ever needed to fight mice on their own terf?
+Do YOU need someone to save you when the air is poisoned but nobody can help?
+Do _YOU_ need something to do once your soul is free floating and ready for harvest?
+
+Never fear! Nanotransen Reaserch and Devopment brings you a breakthrough in both biotech and anomaly studies! 
+THE GNOME! The best helper that botany can ask for! being small enough to get into those TIGHT spaces to do
+tough jobs we gaurentee you will be happy* for their help.
+
+(*Any defects are the responsibility of the station botanist)
+(rumors of gnomes commiting Homicide are false and defamitory. There is no Gnome king)
+
+# Overview
+This design document adds a crafty and fragile ghost role tied to botany.
+
+## Gnome features
+
+ They are a ghost role For early to late round use,
+ they have the following abilities and traits:
+ - A hand slot to pick up items
+ - they can drag items for storage (such as a survival box or backpack)
+ - they can be picked up and worn as hats and die when thrown
+ - they are healed and revived with glue
+ - Gnomes are grown from a Gnome hat that can be planted into a gnome plant that grows the gnome
+ - they are very fragile (10 health) but dont need air and are resistant to tempature damage 
+ - being hollow they are weak to pressure changes that cause them to break quickly, space is an instant death.
+ - can be gibbed into a gnome shard and glass to plant another gnome plant
+ - has a gnome accent that replaces all "no" at the start of words to "GNO" and certain past tence words to GNOMED eg: killed > GNOMED (The captain is GNOMED)
+ - weak to blunt, slightly resistant to slash and peircing, and very resistant to burns
+ - rooms can be "Gnome proofed" with mouse traps
+ - Gnomes act like a normal garden gnome untill possesed by a ghost player
```

Comment:
formatting is odd here since after the first letter after each entry should be upper case

Make them unable to drag, or AT LEAST make sure draggin bodies is near impossible since gnomes shouldnt be able to be paramedics 

The gnome shard seems a little bloaty, just make the mushroom hat give back the seed

Do they have a gnomed scream? would be fun is the had the typical "OO" sound from the gnomed meme but thats prob copyrighted, if you find one that isnt copyrighted and fits it would be a nice detail